README.md

```diff
@@ -19,6 +19,11 @@ If you have an idea how to improve our MVVM Framework, please contact us via our
 
 # Release Notes
 
+### 22.1.3
+* [T1079515](https://supportcenter.devexpress.com/internal/ticket/details/T1079515) - The CompositeCommand and ConfirmationBehavior can work with commands that accept specific type arguments.
+* [Notification Service](https://docs.devexpress.com/WPF/18138/mvvm-framework/services/predefined-set/notificationservice#create-a-native-windows-1011-notification) - you can use the optional **Id** parameter of the [NotificationService.CreatePredefinedNotification](https://docs.devexpress.com/WPF/DevExpress.Mvvm.UI.NotificationService.CreatePredefinedNotification(System.String-System.String-System.String-System.Windows.Media.ImageSource-System.String)) method to identify the notification with which the user interacts.
+* [Module Manager](https://docs.devexpress.com/WPF/118618/mvvm-framework/mif/module-manager#regions) - you can use our new `ModuleInjection.GetRegions` method to get all the regions from a particular ViewModel. 
+
```

Comment:
Lines 25, 28, 29, 30, 34 (below):
Please add a period at the end of these lines.

Line 69:
`* [CompositeCommandBehavior](https://docs.devexpress.com/WPF/18124/mvvm-framework/behaviors/predefined-set/compositecommandbehavior) - our new CanExecuteCondition property specifies whether the command target should be disabled when the CanExecute method of one of the commands returns true.`

Please change "our" to "Our".

Line 72:
`The [DXBinding, DXEvent, and DXCommand](https://docs.devexpress.com/WPF/115770/mvvm-framework/dxbinding) extensions use a new expression evaluation engine by default.`

Please update this sentence without using "by default".

`The new Expression Evaluation Engine provides the following features:`

Can you change "provides" to "includes"?

Lines 85 and 88:
Please change "nuget package version" to "NuGet package version".